### DISCUSSION TITLE
Possibility to cast a dashboard to a media player while MA is playing

### DISCUSSION THREAD

**Discussion by WebfinityDesign** [music-assistant/support#1036] [Feature requests and ideas]
I use MA every day as a wake-up alarm to play a radio station on my television in the bedroom (by lack of any other speaker).
It would be nice to have a dashboard with i.e. today's weather and/or news on the screen at the same time. Now I only have the text: Music Assistant is playing

**Comment by OzGav** [comment]
Slight improvement with MA V2 in that you now get the artist and track name sent to the display as well. The rest of your feature request is noted!

**Reply by WebfinityDesign** [reply]
Any progress on this?
